build.gradle

```diff
@@ -111,6 +111,11 @@ project.afterEvaluate {
                     <option name="lenBefore" value="2" />
                 </LanguageOptions>
                 """.stripIndent()))
+            copyrightManager.append(new XmlParser().parseText("""
```

Comment:
Unfortunately the XmlParser's parseText expects a singular XML tree, so we can't parse everything in one shot (the parser will complain about it not being well-formed if we put both blocks in one string).

We could extract the parser creation (though I'm not sure if it's worth it).